controllers/main_controller.py

```diff
@@ -150,8 +150,7 @@ def _render(self, *args, **kw):
         effective_season_year = SeasonHelper.effective_season_year()
 
         self.template_values.update({
-            'endbuild_datetime_est': SeasonHelper.stop_build_datetime_est(effective_season_year),
-            'endbuild_datetime_utc': SeasonHelper.stop_build_datetime_utc(effective_season_year),
```

Comment:
We should be able to kill `stop_build_datetime_est` and `stop_build_datetime_utc` then, right? It's all dead code now.